index.html

```diff
@@ -4329,6 +4409,46 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.4.0</b> &mdash; <small><i>Apr. 8, 2017</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.3.3...1.4.0">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.4.0/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        Added support for <tt>ES6 classes</tt> with a <tt>preinitialize</tt> method
+      </li>
+      <li>
+        Added <tt>iterator</tt> support implementing JS Iterable protocol
+      </li>
+      <li>
+        Fixed a bug with the hash generation from Router.navigate
+      </li>
+      <li>
+        Use the public <b>on</b> method when listening
```

Comment:
Add a sentence here about how this allows it to play nice with other event libraries (including Node's)